offline-environment.md

```diff
@@ -0,0 +1,63 @@
+# Installing OpenFisca in an offline environment
+
+Some people need to install OpenFisca on a server which has no Internet access.
+Here is a recipe explaining how to do.
+
+The big picture: download Python packages on a machine with Internet access, copy them to the server and install them in a [virtualenv](https://virtualenv.pypa.io/en/stable/).
+
+We assume that it is possible to copy files to the server, ie via an USB key. Or perhaps the server filters only outgoing connections, but accepts incoming connections allowing to copy the files.
+
+## On a machine with Internet access
+
+We are going to create a first virtualenv in which we'll use `pip` to download the `.whl` files in a specific directory.
+
+Here we use [pew](https://github.com/berdario/pew) to simplify virtualenv management.
+
+```sh
+pip install pew
+pew new openfisca-packages
+
+# Upgrade pip itself
+pip install --upgrade pip
+pip --version
+# Should print at least 9.0 at the time we write this doc.
+
+mkdir /tmp/openfisca-packages
+cd /tmp/openfisca-packages
```

Comment:
Is this also wrapped by `virtualenv`? If not, then you should rely on `/tmp` being writable to the user.